week-3/B-array-some/exercise.js

```diff
@@ -12,6 +12,19 @@ var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 // https://nodejs.org/api/process.html#process_process_exit_code
 // process.exit(1);
 
+function isNull(item) {
+  return item === null;
+}
+
+function nullCheck() {
+  if (pairsByIndex.some(isNull)) {
+    console.log("Error! Null value");
+    process.exit(1);
+  }
+}
+
+console.log(nullCheck());
```

Comment:
Well done! This one was a bit tricky, so please try and help others who might struggle if you have time :)

Also, are you sure this console.log is necessary?

Reply:
Quite right. I'll amend it. Cheers!

Reply:
Also, thank you for letting me know it was the right thing to do! The exercise wasn't 100% clear and I put this up as just my best guess.